package.json

```diff
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "@octokit/fixtures": "^3.0.0",
+    "bluebird": "^3.5.1",
```

Comment:
we will removed custom Promise support (see https://github.com/octokit/node-github/pull/623). You shouldn't need to add it back. If it's used in one of the examples, remove the `Promise` option instead

Reply:
I'd also remove the `testPromiseQ.js` file altogether